### PR TITLE
Add content for functions/slopes-lines-curves

### PR DESCRIPTION
**Pull request by razvand**
Add actual content to the skeleton of functions/slopes-lines-curves.

### CODE REVIEW

File: chapters/functions/slopes-lines-curves/reading/README.md [hunk 1]
```diff
@@ -1 +1,114 @@
...
+Linear functions produce straight-line graphs.
+In general, a straight line follows the following equation:
+
+$$y = a + bx$$
+
+where $a$ and $b$ are fixed numbers.
+
+The line on the graph is the set of points:
+
+$$\left \\{ (x,y): x,y \in \mathbb{R}, y = a+bx\right \\}$$
```
Comment (teodutu) [inline chapters/functions/slopes-lines-curves/reading/README.md]:
```suggestion
$$\left \{ (x,y): x,y \in \mathbb{R}, y = a+bx\right \}$$
```

**Review by teodutu — CHANGES_REQUESTED** [review]

File: chapters/functions/slopes-lines-curves/reading/README.md [hunk 2]
```diff
@@ -1 +1,114 @@
...
+Linear functions produce straight-line graphs.
+In general, a straight line follows the following equation:
+
+$$y = a + bx$$
+
+where $a$ and $b$ are fixed numbers.
+
+The line on the graph is the set of points:
+
+$$\left\\\{ (x,y): x,y \in \mathbb{R}, y = a+bx \right\\\}$$
```
Comment (teodutu) [inline chapters/functions/slopes-lines-curves/reading/README.md]:
```suggestion
$$\left\{ (x,y): x,y \in \mathbb{R}, y = a+bx \right\}$$
```

**Review by teodutu — CHANGES_REQUESTED** [review]

File: chapters/functions/slopes-lines-curves/reading/README.md (hunk 2 above)
Comment (razvand) [reply chapters/functions/slopes-lines-curves/reading/README.md]: @teodutu , are you sure about this? If I don't also escape the backslash, there is a complaint of rendering on [GitHub](https://github.com/razvand/ccas/blob/functions/slopes-lines-curves/chapters/functions/slopes-lines-curves/reading/read.md).

Comment (teodutu) [reply chapters/functions/slopes-lines-curves/reading/README.md]: If you do escape it, it doesn't render properly with Docusaurus. Pick your poison.

Comment (razvand) [reply chapters/functions/slopes-lines-curves/reading/README.md]: I replaced curly brackets (`{`, `}`) with square brackets (`[`, `]`) that work on both GitHub and Docusaurus.

Comment (teodutu) [reply chapters/functions/slopes-lines-curves/reading/README.md]: Ok, let's keep it like this to have a working solution on both platforms, but keep in mind that it's wrong to use square brackets for sets. They are used for intervals, while more generic sets are defined using curly brackets.

Comment (teodutu) [reply chapters/functions/slopes-lines-curves/reading/README.md]: It doesn't work on Docusaurus when I build it. However, this works (without escaping `[`, `]`):
```
$$\left [ (x,y): x,y \in \mathbb{R}, y = a+bx \right ]$$
```
This is what is rendered when the square brackets are escaped:
image

**Review by teodutu — APPROVED** [review]
Merry Christmas, @razvand!

**Pull request merged**